app/services/create_account.rb

```diff
@@ -29,11 +30,38 @@ def create_tenant
     Apartment::Tenant.create(account.tenant) do
       initialize_account_data
       account.switch do
-        AdminSet.find_or_create_default_admin_set_id
+        create_defaults
+        fillin_translations
+        add_initial_users
+        true
       end
     end
   end
 
+  def create_defaults
+    Hyrax::CollectionType.find_or_create_default_collection_type
+    Hyrax::CollectionType.find_or_create_admin_set_type
+    AdminSet.find_or_create_default_admin_set_id
+  end
+
+  # Workaround for upstream issue https://github.com/samvera/hyrax/issues/3136
+  def fillin_translations
+    collection_types = Hyrax::CollectionType.all
+    collection_types.each do |c|
+      next unless c.title =~ /^translation missing/
+      oldtitle = c.title
+      c.title = I18n.t(c.title.gsub("translation missing: en.", ''))
+      c.save
+      Rails.logger.debug "#{oldtitle} changed to #{c.title}"
+    end
+  end
+
+  def add_initial_users
+    users.each do |user|
```

Comment:
I'm doing some manual testing and this line is throwing a `NoMethodError` because `users` is nil.

Reply:
The suggestion in https://github.com/samvera/hyku/pull/1641/files#r482229319 fixed this for me.

Reply:
thanks @cjcolvar this makes sense